sdk/storage/storage-datalake/package.json

```diff
@@ -41,9 +50,30 @@
     "tsconfig.json"
   ],
   "scripts": {
-    "build": "tsc && rollup -c rollup.config.js && npm run minify",
-    "minify": "uglifyjs -c -m --comments --source-map \"content='./dist/storage-datalake.js.map'\" -o ./dist/storage-datalake.min.js ./dist/storage-datalake.js",
-    "prepack": "npm install && npm run build"
+    "audit": "node ../../../common/scripts/rush-audit.js && rimraf node_modules package-lock.json && npm i --package-lock-only 2>&1 && npm audit",
+    "build:autorest": "autorest ./swagger/README.md --typescript --use=@microsoft.azure/autorest.typescript@2.2.1",
+    "build:browserzip": "echo skipped",
+    "build:es6": "tsc -p tsconfig.json",
+    "build:nodebrowser": "rollup -c 2>&1",
+    "build:test": "echo skipped",
+    "build": "npm run build:es6 && npm run build:nodebrowser && npm run build:browserzip",
+    "check-format": "prettier --list-different --config ../../.prettierrc.json \"src/**/*.ts\" \"test/**/*.ts\" \"*.{js,json}\"",
+    "clean": "rimraf dist dist-esm dist-test typings temp browser/*.js* browser/*.zip statistics.html coverage coverage-browser .nyc_output *.tgz *.log test*.xml TEST*.xml",
+    "extract-api": "tsc -p . && api-extractor run --local",
```

Comment:
Does this target actually work here? Just curious

Reply:
the extract-api command doesn't work. Should I just skip it?

Reply:
removed this step